### PR TITLE
CNDE-1849 Hot fix for tombstone message calls

### PR DESCRIPTION
**Pull request by sveselev**
## Notes

Fix for tombstone (clearing) message call order.

## JIRA

- **Related story**: [CNDE-1849](https://cdc-nbs.atlassian.net/browse/CNDE-1849)

## Checklist

- [x] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [x] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [x] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [x] Bugfix
- [ ] New feature
- [ ] Breaking change

## Testing

- [x] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [x] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java [hunk 1]
```diff
@@ -51,13 +51,13 @@ public class ProcessInvestigationDataUtil {
...
 
-        InvestigationTransformed investigationTransformed = new InvestigationTransformed();
+        InvestigationTransformed investigationTransformed = new InvestigationTransformed(investigation.getPublicHealthCaseUid());
 
         transformPersonParticipations(investigation.getPersonParticipations(), investigationTransformed);
         transformOrganizationParticipations(investigation.getOrganizationParticipations(), investigationTransformed);
         transformActIds(investigation.getActIds(), investigationTransformed);
         transformObservationIds(investigation.getObservationNotificationIds(), investigationTransformed);
-        transformInvestigationConfirmationMethod(investigation.getInvestigationConfirmationMethod());
+        transformInvestigationConfirmationMethod(investigation.getInvestigationConfirmationMethod(), investigationTransformed);
```
Comment (upasanapattnaik-eq) [inline investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java:60]: Does this have a null check?

Comment (alog-enquizit) [reply investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java:60]: Line 54 might already have initialized the object and we might not need a NPE check here..

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

File: investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java (hunk 1 above)
Comment (sveselev) [reply investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java:60]: Every nested data is checked inside the corresponded method.

**Pull request merged**